Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -191,4 +196,31 @@ namespace AudioCommon
 		g_sound_stream->GetMixer()->StopLogDSPAudio();
 		s_audio_dump_start = false;
 	}
+
+	void IncreaseVolume(unsigned short offset)
+	{
+		SConfig::GetInstance().m_IsMuted = false;
+		int& currentVolume = SConfig::GetInstance().m_Volume;
+		currentVolume += offset;
+		if (currentVolume > AUDIO_VOLUME_MAX)
+			currentVolume = AUDIO_VOLUME_MAX;
+		UpdateSoundStream();
+	}
+
+	void DecreaseVolume(unsigned short offset)
+	{
+		SConfig::GetInstance().m_IsMuted = false;
+		int& currentVolume = SConfig::GetInstance().m_Volume;
+		currentVolume -= offset;
+		if (currentVolume < AUDIO_VOLUME_MIN)
+			currentVolume = AUDIO_VOLUME_MIN;
```

Comment:
I'm fairly sure there's no real reason. Setting the volume to 0 in the config and starting up dolphin always worked fine. It's most likely just a simple oversight.